pkg/cmd/scan/output/console.go

```diff
@@ -93,12 +95,20 @@ func (c *Console) Write(analysis *analyser.Analysis) error {
 
 	c.writeSummary(analysis)
 
+	policy := false
 	for _, alerts := range analysis.Alerts() {
 		for _, alert := range alerts {
-			fmt.Printf("%s\n", color.YellowString(alert.Message))
+			fmt.Printf("%s\n", color.YellowString(alert.Message()))
+			if _, ok := alert.(remote.EnumerationAccessDeniedAlert); ok {
```

Comment:
Hm i remember this remark but my statement set policy to true if we have EnumerationAccessDeniedAlert, and dont touch policy var if Alert is not of this type.
So this can be a problem if EnumerationAccessDeniedAlert dont give 403 status code but imo HandleResourceEnumerationError doesn't create Alert of type EnumerationAccessDeniedAlert if 403 StatusCode statement isn't true.
In this configuration, my loop can't fail, but the best i can do to prove this is create a test with some kinds of different Alert type.
However, you must be right, so i dont understand how to verify that if my EnumerationAccessDeniedAlert should set my policy var to true or not.

Reply:
You are right my bad, it was a lack of attention for me this time 😓